docs/en/engines/table-engines/integrations/postgresql.md

```diff
@@ -0,0 +1,102 @@
+---
+toc_priority: 8
+toc_title: PostgreSQL
+---
+
+# PosgtreSQL {#postgresql}
+
+The PostgreSQL engine allows you to perform `SELECT` queries on data that is stored on a remote PostgreSQL server.
+
+## Creating a Table {#creating-a-table}
+
+``` sql
+CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
+(
+    name1 [type1] [DEFAULT|MATERIALIZED|ALIAS expr1] [TTL expr1],
+    name2 [type2] [DEFAULT|MATERIALIZED|ALIAS expr2] [TTL expr2],
+    ...
+) ENGINE = PostgreSQL('host:port', 'database', 'table', 'user', 'password');
+```
+
+See a detailed description of the [CREATE TABLE](../../../sql-reference/statements/create/table.md#create-table-query) query.
+
+The table structure can differ from the original PostgreSQL table structure:
```

Comment:
Also it can be mentioned that there is a setting `external_table_functions_use_nulls`. Default is true, if false - table function will not make nullable columns and will insert default values instead of nulls.

Reply:
fixed in https://github.com/ClickHouse/ClickHouse/pull/21078/commits/aacc9c091f983454faa04526b13b9792006684b6